projects/plugins/jetpack/functions.global.php

```diff
@@ -50,8 +50,10 @@ function jetpack_deprecated_function( $function, $replacement, $version ) { // p
 	) {
 		error_log( // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_error_log
 			sprintf(
-				/* Translators: 1. Function name. 2. Jetpack version number. */
-				__( 'The %1$s function will be removed from the Jetpack plugin in version %2$s.', 'jetpack' ),
+				did_action( 'init' ) ?
```

Comment:
While Core's message says "init", the actual check it does is more like
```suggestion
				doing_action( 'after_setup_theme' ) || did_action( 'after_setup_theme' ) ?
```